### DISCUSSION TITLE
[help] My 6 xch coins are missing

### DISCUSSION THREAD

**Discussion by fengyezhiwu** [Chia-Network/chia-blockchain#5929] [Support]
My 6 xch coins are missing?
![d47bf73f67ad363dbc390e388ffba68](https://user-images.githubusercontent.com/84707525/119324214-2a036c00-bcb2-11eb-94be-3dfa90f4c461.jpg)
![a8018abd89e293d421929ebacf7a637](https://user-images.githubusercontent.com/84707525/119324241-338cd400-bcb2-11eb-9960-84f520c61395.jpg)
![7d6a8c6e1ea21163c44bc05e8669670](https://user-images.githubusercontent.com/84707525/119324576-93837a80-bcb2-11eb-9f42-996f05a9d04f.jpg)

address:xch1q3d9np83g3uur2jq65j06xqtnvnjmhq8cr2svkdjvj27rl90l6ps6lvf4v

**Comment by metoos003** [comment]
UPDATA 1.1.6

**Reply by fengyezhiwu** [reply]
is 1.1.6

**Reply by fengyezhiwu** [reply]
What used to be normal suddenly disappeared

**Comment by fengyezhiwu** [comment]
I farmed on my own

**Comment by fengyezhiwu** [comment]
My wallet matches the explorer in that that address had 6 coins and they have all been transferred.xch13jzm94939qaamxqyy0he8pgkhg4nvmj6h6nhvqeek02jqgd9f8jslk7mds,Where is the address。No movement.What should I do to get it back。

**Comment by BrandtH22** [comment]
@fengyezhiwu , unfortunately it appears someone has gained access to your keyset and/or device and stolen your funds. Since chia is a truly decentralized network there is no way to reclaim the funds or revert the transaction(s). One thing we recommend is ensuring that you and your farm are following security best practices to reduce malicious actors ability to gain access to your keys and/or device, many of these best practices are outlined here: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

Since we unfortunately cannot do more to assist with the stolen funds we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
